srv/components/misc_pkgs/kafka/teardown.sls

```diff
@@ -35,6 +35,14 @@ Remove zookeeper log directory:
   file.absent:
     - name: /var/log/zookeeper
 
+Remove accepted epoch:
```

Comment:
Better cleanup entire /var/lib/zookeeper

Reply:
@sachinpunadikar 
We are doing that already on line 32 above.
Good catch. We don't need this change. There's something else we need to check.